firmware/src/tasks/task_health_monitor.cpp

```diff
@@ -109,18 +110,19 @@ static void init_communication();
 
     // Beep out ready buzzer
     if ((m_fsm_enum == READY) && (ready_timer >= 500)) {
-      buzzer_queue_status(CATS_BUZZ_READY);
+      m_task_buzzer.Beep(Buzzer::BeepCode::kReady);
       ready_timer = 0;
     } else if (m_fsm_enum == READY) {
       ++ready_timer;
     }
 
     // Beep out transitions from moving to ready and back
-    if (m_fsm_enum == READY && fsm_updated) buzzer_queue_status(CATS_BUZZ_CHANGED_READY);
-    if (m_fsm_enum == MOVING && fsm_updated) buzzer_queue_status(CATS_BUZZ_CHANGED_MOVING);
-
-    // Update the buzzer
-    buzzer_handler_update();
+    if (m_fsm_enum == READY && fsm_updated) {
+      m_task_buzzer.Beep(Buzzer::BeepCode::kChangedReady);
+    }
+    if (m_fsm_enum == MOVING && fsm_updated && tick_count > 100U) {
```

Comment:
why do we need this tick_count?

Reply:
Because we trigger moving right after bootup and I dont want it to beep that transition